examples/instances/main.tf

```diff
@@ -2,7 +2,9 @@ terraform {
   required_providers {
     bytebase = {
       version = "0.0.3"
-      source  = "terraform.local/bytebase/bytebase"
+      # The source is only used for local development.
+      # To use it in production, please replace it with "registry.terraform.io/bytebase/bytebase"
+      source = "terraform.local/bytebase/bytebase"
```

Comment:
Maybe this should be reversed, the comment is `terraform.local/bytebase/bytebase` while the default is `registry.terraform.io/bytebase/bytebase`.

Since development usage is rare.

Reply:
LGTM, updated